client/components/data/query-products-list/index.jsx

```diff
@@ -16,13 +16,19 @@ const request =
 		dispatch( requestProductsList( props ) );
 	};
 
-export function useQueryProductsList( { type = 'all', currency, persist } = {} ) {
+export function useQueryProductsList( {
+	type = 'all',
+	currency,
+	persist,
+	product_slug_list,
```

Comment:
Since `useQueryProductsList` is in the "Calypso world", I think this param should be in camel case (`productSlugList`). Same for `QueryProductsList` below.

I was also wondering if it's possible to convert this file to TS. It will offer better DX for developers using this query/component.

Reply:
I could convert it to TS but would prefer to do so in another PR once this is deployed

Reply:
Sounds good, thanks!